hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/mapreduce/TestHFileOutputFormat2.java

```diff
@@ -554,18 +554,43 @@ private byte[][] generateRandomStartKeys(int numKeys) {
     // first region start key is always empty
     ret[0] = HConstants.EMPTY_BYTE_ARRAY;
     for (int i = 1; i < numKeys; i++) {
-      ret[i] =
-        PerformanceEvaluation.generateData(random, PerformanceEvaluation.DEFAULT_VALUE_LENGTH);
+      ret[i] = generateData(random, DEFAULT_VALUE_LENGTH);
     }
     return ret;
   }
 
+  /*
```

Comment:
This is trivial enough that duplicating it is no big deal, but maybe we could move this to LoadTestUtil (if we move that to hbase-commons)

Reply:
Makes sense.

Reply:
I have tried to make local copies of such variable wherever i could find shared usage